Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -294,3 +294,126 @@ angular.module("umbraco").controller("customBlockController", function ($scope,
     });    
 });
 ```
+
+## Creating Blocklist programmatically
+
+In this example we are creating some Blocklist objects under the `People` property in `Home` Document Type. The `People` property implements a Blocklist datatype where a `Person` Document Type can be created. The `Person` Document Type has two properties - `user_name` and `user_email`.
+
+The approach to saving Blocklist content programmatically is similar to Nested Content - though the JSON schema is a bit different.
+
+The JSON object we will pass into the `People` property will look like this:
+
+```json
+{
+   "layout":{
+      "Umbraco.BlockList":[
+         {
+            "contentUdi":"umb://element/e28d73b5b9394631b7184c5cd6065fed"
+         },
+         {
+            "contentUdi":"umb://element/e51a247138594f6ca48f541d1bc1e09f"
+         }
+      ]
+   },
+   "contentData":[
+      {
+         "contentTypeKey":"68075bd7-bd89-4614-9157-1979dba05768",
+         "udi":"umb://element/e28d73b5b9394631b7184c5cd6065fed",
+         "user_name":"Test user",
+         "user_email":"test@test.com"
+      },
+      {
+         "contentTypeKey":"68075bd7-bd89-4614-9157-1979dba05768",
+         "udi":"umb://element/e51a247138594f6ca48f541d1bc1e09f",
+         "user_name":"John",
+         "user_email":"john@johnindustries.com"
+      }
+   ],
+   "settingsData":[
```

Comment:
Is it worth having a basic example for settingsData or at least a comment that explains this is how data is saved for the 'settings' for the Block List item?

Reply:
Most definitely! Unfortunately I do not know how to get the settings right quite yet. I was planning to update that later